ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1252,6 +1255,40 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           </ModalFooter>
         </Modal>
         }
+        {showConfirmationModal &&
+          <Modal width={500}>
+            <ModalTitle style={{fontSize: '16px', marginBottom: 0}}>
+              {this.renderButtonText()}
+            </ModalTitle>
+            <ModalBody style={{color: colors.primary, lineHeight: '1rem', marginTop: '0.25rem'}}>
+              <div>Your responses to these questions:</div>
+              <div style={{margin: '0.25rem 0 0.25rem 1rem'}}>
+                <span style={{fontWeight: 600}}>Primary purpose of your project</span> (Question 1)<br/>
```

Comment:
I believe `font-weight` is inherited - can you put it on the parent div instead of on each child?

Reply:
Leaving as is since the text after each `<span>` (Question 1, Question 2...) needs to have normal `font-weight`

Reply:
ah, I get it! Misread before. This is fine by me